MC/config/QC/json/its-mc-tracks-qc.json

```diff
@@ -0,0 +1,65 @@
+{
+  "qc" : {
+    "config" : {
+      "database" : {
+        "implementation" : "CCDB",
+        "host" : "ccdb-test.cern.ch:8080",
+        "username" : "not_applicable",
+        "password" : "not_applicable",
+        "name" : "not_applicable"
+      },
+      "Activity" : {
+        "number" : "42",
+        "type" : "2",
+        "provenance": "qc_mc",
+        "passName": "passMC",
+        "periodName": "SimChallenge"
+      },
+      "monitoring" : {
+        "url" : "infologger:///debug?qc"
+      },
+      "consul" : {
+        "url" : "http://consul-test.cern.ch:8500"
```

Comment:
Do you suggest to delete "consul" and "monitoring"?

Reply:
Consul is not used in MC. @knopers8 knows for sure what should definitely be removed, and what can stay and won't make issues, but in general, it would be good to have kind of "clean" json - maybe we should check all of them.

Reply:
Monitoring is fine as it is. Consul url should be indeed left empty (`""`)